src/pyedb/dotnet/database/sim_setup_data/data/mesh_operation.py

```diff
@@ -47,6 +47,15 @@ def __init__(self, parent, edb_object):
             "kNumMeshOpTypes": self._edb_object.TMeshOpType.kNumMeshOpTypes,
         }
 
+    @property
+    def net_layer_info(self):
+        """Adding property for grpc compatibility."""
```

Comment:
Can you also copy the docstring from grpc?

Reply:
Done